client/src/components/Reviews/ProductBreakdown.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { VscTriangleDown } from 'react-icons/vsc';
+import ClickTracker from './ClickTracker';
 
-const ProductBreakdown = ({currentProductMeta}) => {
+const ProductBreakdown = ({currentProductMeta, handleTrackingClick}) => {
 
   const { characteristics } = currentProductMeta;
   const charsToPercent = {};
+  const widget = 'Ratings and Reviews';
```

Comment:
i hardcoded my widget name inside the onClick={....} block. i wonder what the up and downsides are to both approaches here.

Reply:
I like this idea, I made the change in mine.